tools/python/liveanalytics_migration_scripts/targets/timestream_for_influxdb/transform/transform.py

```diff
@@ -503,12 +523,21 @@ def translate_athena_table_to_line_protocol(
                 lp_translation_query += f"""
                 CASE WHEN \"{measure_value_name}\" IS NOT NULL THEN '{measure_value_name}="' || CAST(\"{measure_value_name}\" AS VARCHAR) || '",' ELSE '' END {delimiter}
                 """
+            elif measure_value_type == "timestamp":
+                if use_ns_precision:
+                    lp_translation_query += f"""
+                    CASE WHEN \"{measure_value_name}_ns\" IS NOT NULL THEN '{measure_value_name}=' || {measure_value_name}_ns || ',' ELSE '' END {delimiter}
```

Comment:
```suggestion
                    CASE WHEN \"{measure_value_name}_ns\" IS NOT NULL THEN '{measure_value_name}=' || 'i' || {measure_value_name}_ns || ',' ELSE '' END {delimiter}
```
This will make sure the timestamp is added as an integer and not a double.

We should also do this for other measure values that are type `INT` or `BIGINT`.

Reply:
does this actually matter, since it will just be in text format? in what scenarios might we get a timestamp as a double?

Reply:
If I understand how this process works, `{measure_value_name}_ns` will resolve to a `VARCHAR` value containing an integer. In line protocol, a numerical value without a preceding `i` is treated as a double for fields.